tests/fakedroid.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+# This is a script to run "on-device" tests in CI, without the device.
+# Takes the bootstrap aarch64 zipball, unpacks, proots into it.
+# This won't catch all bugs, of course, but that's something.
+
+# Set up envvars, prepare directories:
+
+set -uexo pipefail
```

Comment:
Could you remove the -x or at least make it optional? I tested it locally and it is producing a lot of noise when you are not interested in changing the fakedroid.sh script.

Reply:
Sure, added